content/sensu-core/2.0/reference/entities.md

```diff
@@ -14,21 +14,21 @@ menu:
+An Entity, formally known as a client in Sensu 1.x, represents anything (e.g. server, container, network switch, etc) that needs to be monitored. It can contain system information such as the hostname, OS, platform, and version. An Entity not only provides context to event data (what/where the event is from) but an event's uniqueness is determined by the check name and the ID of the entity upon which the check ran.
 
 ## Agent entities
 
 ### What is an agent entity?
 
-An agent entity is a Sensu entity which falls under the "agent" class. Agent entities are monitoring agents, which are installed and run on every system that needs to be monitored. The entity is responsible for registering the system with Sensu, sending keepalive messages (the Sensu heartbeat mechanism), and executing monitoring checks. Each entity is a member of one or more subscriptions – a list of roles and/or responsibilities assigned to the system (e.g. a webserver, database, etc). Sensu entities will "subscribe" to (or watch for) check requests published by the Sensu server (via the Sensu Transport), execute the corresponding requests locally, and publish the results of the check back to the transport (to be processed by a Sensu server).
+Agent entities are monitoring agents, which are installed and run on every system that needs to be monitored. The entity is responsible for registering the system with Sensu, sending keepalive messages (the Sensu heartbeat mechanism), and executing monitoring checks. Each entity is a member of one or more subscriptions – a list of roles and/or responsibilities assigned to the system (e.g. a webserver, database, etc). Sensu entities will "subscribe" to (or watch for) check requests published by the Sensu server (via the Sensu Transport), execute the corresponding requests locally, and publish the results of the check back to the transport (to be processed by a Sensu server).
 
 ### Creating an agent entity
 
-To create an agent entity, Sensu 2 must be installed. Upon spinning up the sensu-agent, you can use several flags to customize the properties of the entity. In the example below, the optional flags are used to describe the default values of the agent entity that gets created upon starting sensu-agent. Other flags/properties you can set for your agent entity include: --deregister, --subscriptions, --deregistration-handler, and --redact.
+To create an agent entity, Sensu 2.x must be installed. Upon spinning up the sensu-agent, you can use several flags to customize the properties of the entity. In the example below, the optional flags are used to describe the default values of the agent entity that gets created upon starting sensu-agent. Other flags/properties you can set for your agent entity include: --deregister, --subscriptions, --deregistration-handler, and --redact.
```

Comment:
When talking about the agent, it's best to refer to it as the agent or mention the package `sensu-agent` with syntax highlighting. Bonus points for highlighting the flags as well. 

Side note - will these be consistent enough that you want to list them or can we link to where they will be kept up to date?